DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.1.0-preview1-007041
+2.0.2-vspre-006963
```

Comment:
This means we're no longer using a master version of the SDK with the master version of the CLI?  Shouldn't we merge release/2.0-vs and release/2.0.0 into master in the SDK, then use that version of the SDK here?  Or can we not do that right now because signing is down?

Reply:
> Or can we not do that right now because signing is down?

I think that's exactly the issue. @livarcocc mentioned as much in  standup today and note that f1fe4af does this deliberately post-merge.

Reply:
Yep, signing.